apps/analysis/serializers.py

```diff
@@ -9,6 +9,7 @@
     NestedCreateMixin,
     NestedUpdateMixin
 )
+from django.conf import settings
```

Comment:
Let's move this before `from rest_framework...`

Reply:
great!!!